README.md

```diff
@@ -79,6 +79,56 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
+
+Follow these steps to create your first project using Corral.
+
+1. Create an empty folder and switch to this directory. This will be our example project to use Corral
+
+```bash
+mkdir myproject
+cd myproject
+```
+
+2. Initialize Corral. It will create `corral.json` and `lock.json` files. At this moment they won't have much information since you haven't added any dependency yet.
```

Comment:
Another small point here, it reads strange to me to say "any dependency" rather than "a dependency" or "any dependencies" -- is it just me stumbling on this wording?

Reply:
fair point. "any dependencies" seems more reasonable

Reply:
good one. i missed that.